api/src/main/java/org/apache/cloudstack/api/command/user/firewall/CreatePortForwardingRuleCmd.java

```diff
@@ -358,7 +358,6 @@ public void create() {
             setEntityId(result.getId());
             setEntityUuid(result.getUuid());
         } catch (NetworkRuleConflictException ex) {
-            s_logger.info("Network rule conflict: ", ex);
             s_logger.trace("Network Rule Conflict: ", ex);
             throw new ServerApiException(ApiErrorCode.NETWORK_RULE_CONFLICT_ERROR, ex.getMessage());
```

Comment:
What about using ` throw new ServerApiException(ApiErrorCode.NETWORK_RULE_CONFLICT_ERROR, ex);` instead of what we have here?